package.json

```diff
@@ -46,6 +47,7 @@
     "gulp-rename": "^1.2.0",
     "gulp-uglify": "^1.1.0",
     "istanbul": "^0.3.5",
+    "jayson": "^1.2.2",
```

Comment:
Oh, I was thinking we would expose it from index.js, but maybe add a comment that says "experimental". We might as well add "experimental" to RippleAPIBroadcast while we're at it too.